### PR TITLE
#20 Change a condition in computePositionTopFromRect

### PR DESCRIPTION
**Pull request by RyoNkmr**
# DONE

- refactor for flow
- fix bug

### CODE REVIEW

File: src/sticky.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default class Sticky {
     return this.$$wrapper;
   }
 
-  set wrapper(value: SelectorOrElement): void {
+  set wrapper(value?: SelectorOrElement): void {
```
Comment (RyoNkmr) [inline src/sticky.js]: `normalizeElement` fallbacks inside if `value` is empty

File: src/sticky.js [hunk 2]
```diff
@@ -179,28 +182,29 @@ export default class Sticky {
...
-    if (this.isStickToBottom) {
-      if (this.rect.top === this.marginTop) {
-        this.isStickToBottom = false;
-        return;
-      }
-      if (this.rect.top < this.marginTop) {
-        this.top = relativeFloor - this.rect.height;
-        return;
-      }
+    if (!this.isStickToBottom) {
```
Comment (RyoNkmr) [inline src/sticky.js]: early return

File: src/sticky.js [hunk 3]
```diff
@@ -108,12 +108,15 @@ export default class Sticky {
     return absoluteBottom - paddingBottomPixels;
   }
 
-  static normalizeElement(value: SelectorOrElement, ...fallbacks: MaybeHTMLElement[]): HTMLElement {
-    if (value instanceof HTMLElement) {
+  static normalizeElement(
+    value?: SelectorOrElement,
+    ...fallbacks: MaybeHTMLElement[]
+  ): HTMLElement {
+    if (value && value instanceof HTMLElement) {
```
Comment (resessh) [inline src/sticky.js]: 👍

File: src/sticky.js [hunk 4]
```diff
@@ -179,28 +182,29 @@ export default class Sticky {
...
     this.rect = rect;
-    if (this.options.wrapper instanceof HTMLElement) {
-      this.floor = Sticky.computeAbsoluteFloor(this.options.wrapper);
-    }
+    this.floor = Sticky.computeAbsoluteFloor(this.wrapper);
+
     const relativeFloor = (this.floor || 0) - global.pageYOffset;
-    if (this.rect.bottom > relativeFloor && !this.isStickToBottom) {
+
+    if (this.rect.bottom >= relativeFloor && !this.isStickToBottom) {
```
Comment (resessh) [inline src/sticky.js]: 👀

**Pull request merged**